docs/src/main/asciidoc/scheduler-reference.adoc

```diff
@@ -54,15 +54,36 @@ The syntax used in CRON expressions is controlled by the `quarkus.scheduler.cron
 The values can be `cron4j`, `quartz`, `unix` and `spring`.
 `quartz` is used by default.
 
-If a CRON expression starts with `{` and ends with `}` then the scheduler attempts to find a corresponding config property and use the configured value instead. 
+The `cron` attribute supports <<config-reference#using_property_expressions,Property Expressions>> including default values and nested
+Property Expressions. (Note that "{property.path}" style expressions are still supported but don't offer the full functionality of Property Expressions.)
+
 
 .CRON Config Property Example
 [source,java]
 ----
-@Scheduled(cron = "{myMethod.cron.expr}")
+@Scheduled(cron = "${myMethod.cron.expr}")
+void myMethod() { }
+----
+
+If you wish to disable a specific scheduled method, you can set its cron expression to `"off"` or `"disabled"`.
+
+.application.properties
+[source,properties]
+----
+myMethod.cron.expr=disabled
+----
+
+Property Expressions allow you to define a default value that is used, if the property is not configured.
+
+.CRON Config Property Example with default `5m`
+[source,java]
+----
+@Scheduled(cron = "${myMethod.cron.expr:5m}")
```

Comment:
Well, `5m` is not a valid cron expression. I assume that the original intent was to show some valid default value, or?

Reply:
Oh, yes of course.